contracts/cw20-base/src/enumerable.rs

```diff
@@ -27,8 +27,10 @@ pub fn query_all_allowances<S: Storage, A: Api, Q: Querier>(
             let (k, v) = item?;
             Ok(AllowanceInfo {
                 spender: api.human_address(&CanonicalAddr::from(k))?,
-                allowance: v.allowance,
-                expires: v.expires,
+                allowance_response: AllowanceResponse {
```

Comment:
Can't we just do `allowance_response: v`?

Much less verbose and helps make use of flatten